src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -784,6 +789,76 @@ private void PopulateEncLogTableParameters()
             }
         }
 
+        private void PopulateEncLogTableCustomAttributes()
+        {
+            var attributeMap = CreateExistingAttributeMap(_previousGeneration.MetadataReader, _previousGeneration.CustomAttributesAdded, out var lastRow);
```

Comment:
>_previousGeneration.MetadataReader [](http://example.com/codeflow?start=58&length=34)

Should this always be the baseline (0-th generation) rather then the previous one?

Reply:
This is `_previousGeneration.OriginalMetadata.MetadataReader`, but I can make it explicit